i18n/lang_fr_FR/messages.po

```diff
@@ -212,300 +213,303 @@ msgstr ""
 
 #: screens/options.py:265
 msgid "&Fullscreen:"
-msgstr ""
+msgstr "&Plein écran :"
 
 #: screens/options.py:266
 msgid "&Mouse grab:"
-msgstr ""
+msgstr "&Souris captive :"
 
 #: screens/options.py:267
 msgid "Da&y/night display:"
-msgstr ""
+msgstr "Affichage c&ycle jour/nuit :"
 
 #: screens/options.py:268
 msgid "Language:"
-msgstr ""
+msgstr "Langage :"
 
 #: screens/options.py:269
 msgid "Resolution:"
-msgstr ""
+msgstr "Résolution :"
 
 #: screens/options.py:270
 msgid "&CUSTOM:"
-msgstr ""
+msgstr "&Personalisée :"
 
 #: screens/options.py:275 screens/options.py:280 screens/options.py:337
 #: screens/options.py:346 screens/options.py:353 screens/options.py:439
 #: screens/options.py:465
 msgid "YES"
-msgstr ""
+msgstr "OUI"
 
 #: screens/options.py:277 screens/options.py:282 screens/options.py:339
 #: screens/options.py:348 screens/options.py:355 screens/options.py:441
 #: screens/options.py:467
 msgid "NO"
-msgstr ""
+msgstr "NON"
 
 #: screens/options.py:410
 msgid "Sound buffering:"
-msgstr ""
+msgstr "Mise en mémoire tampon du son :"
 
 #: screens/options.py:416
 msgid "&LOW"
-msgstr ""
+msgstr "&BAS"
 
 #: screens/options.py:422
 msgid "&NORMAL"
-msgstr ""
+msgstr "&NORMAL"
 
 #: screens/options.py:428
 msgid "&HIGH"
-msgstr ""
+msgstr "&ÉLEVÉE"
 
 #: screens/options.py:434
 msgid "&Sound:"
-msgstr ""
+msgstr "&Son :"
 
 #: screens/options.py:435
 msgid "GUI Volume:"
-msgstr ""
+msgstr "Son de l’interface :"
 
 #: screens/options.py:436
 msgid "Music Volume:"
-msgstr ""
+msgstr "Volume de la musique :"
 
 #: screens/report.py:67
 msgid "&Midnight"
-msgstr ""
+msgstr "&Minuit"
 
 #: screens/report.py:68
 msgid "24 &Hours"
-msgstr ""
+msgstr "24 &Heures"
 
 #: screens/report.py:79
 msgid "Financial report"
-msgstr ""
+msgstr "Rapport Financier"
 
 #: screens/report.py:83
 msgid "CPU Usage"
-msgstr ""
+msgstr "Usage CPU"
 
 #: screens/report.py:89
 msgid "Current Money:"
-msgstr ""
+msgstr "Compte Courant :"
 
 #: screens/report.py:90
 msgid "Jobs:"
-msgstr ""
+msgstr "Emplois :"
 
 #: screens/report.py:91
 msgid "Research:"
-msgstr ""
+msgstr "Recherches :"
 
 #: screens/report.py:93
 msgid "Construction:"
-msgstr ""
+msgstr "Construction :"
 
 #: screens/report.py:94
 #, python-format
 msgid "Interest (%s):"
-msgstr ""
+msgstr "Intérêts (%s) :"
 
 #: screens/report.py:96
 msgid "Income:"
-msgstr ""
+msgstr "Revenu :"
 
 #: screens/report.py:99
 msgid "Money at Midnight:"
-msgstr ""
+msgstr "Argent à Minuit:"
 
 #: screens/report.py:101
 msgid "Money in 24 hours:"
-msgstr ""
+msgstr "Revenu en 24 heures :"
 
 #: screens/report.py:109
 msgid "Total CPU:"
-msgstr ""
+msgstr "CPU Total :"
 
 #: screens/report.py:110
 msgid "Sleeping CPU:"
-msgstr ""
+msgstr "CPU en veille :"
 
 #: screens/report.py:111
 msgid "Research CPU:"
-msgstr ""
+msgstr "CPU dédié à la recherche :"
 
 #: screens/report.py:112
 msgid "Job CPU:"
-msgstr ""
+msgstr "Travail CPU :"
 
 #: screens/report.py:113
 msgid "CPU pool:"
-msgstr ""
+msgstr "Grappe CPU :"
```

Comment:
Pool ne correspond pas à Grappe. Mais il y a pas vraiment d'équivalent en Francais, alors je ne sais pas.

Reply:
if there's no corresponding word, usually it's the smartest to leave the original english term

Reply:
Yes it's seems that the majority of professional do that. That good for me.

Reply:
Grappe se dis pour pour un pool de machine. Sinon le mot cluster pourrait fonctionner.

Reply:
Oui on est dans la complexité de la traduction. CPU pool renvoie à un groupement de la puissance de calcul restante. Grappe correspond plus à cluster, et donc à un description physique. Grappe est pas très parlant pour renvoyer à la notion de ressource partagé.
Pool semble entré dans les dictionnaires maintenant en tant qu’anglicisme, autant l'utilisé.